packages/core/contracts-ovm/insured-bridge/implementation/BridgePool.sol

```diff
@@ -113,12 +125,14 @@ contract BridgePool is Testable, BridgePoolInterface, ExpandedERC20 {
     constructor(
         address _bridgeAdmin,
         address _l1Token,
+        uint256 _lpFeeRatePerSecond,
         address _timer
     ) Testable(_timer) ExpandedERC20("UMA Insured Bride LP Token", "UMA-LP", 18) {
         bridgeAdmin = BridgeAdminInterface(_bridgeAdmin);
-        require(bridgeAdmin.finder() != address(0), "Invalid bridge pool factory");
 
         l1Token = IERC20(_l1Token);
+        lastLpFeeUpdate = getCurrentTime();
+        lpFeeRatePerSecond = _lpFeeRatePerSecond;
```

Comment:
Should we constrain this %/second at all? I'm curious what's an example value that would be realistic

Reply:
I dont really think constraining this is that important. If this number is really big (like 100% per second) then all the fees are paid out instantly. This implementation limits the maximum payout to the total pending rewards so there is never a situation where we payout more than the total to be paid; it simply modifies how quickly the rewards are paid out.

We would, however, want a modifier to update this. I thought I'd leave this for later though.